src/components/filter-fields/DatePicker2.jsx

```diff
@@ -0,0 +1,278 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { ChevronDown, ChevronLeft, ChevronRight, ChevronUp } from 'react-feather';
+import { useTranslation } from 'react-i18next';
+import 'styles/datepiker2.scss';
+import { useFormik } from 'formik';
+import Yup from 'utils/yup';
+import moment from 'moment';
+import { Form } from '../form-components';
+
+
+const DatePicker2 = (props) => {
+  const { name, onChange, value } = props;
+  const { t } = useTranslation();
+
+  const [isShowDropdown, setShowDropdown] = useState(false);
+  const currentDate = new Date();
+  const [currentYear, setCurrentYear] = useState(currentDate.getFullYear());
+  const [selectValue, setSelectValue] = useState('false');
+  const monthArr = [
+    'January',
+    'February',
+    'March',
+    'April',
+    'May',
+    'June',
+    'July',
+    'August',
+    'September',
+    'October',
+    'November',
+    'December',
+  ];
+
+  const formik = useFormik({
+    initialValues: {
+      day: '',
+      month: '',
+      year: '',
+    },
+    validationSchema: Yup.object({
+      month: Yup
+        .string()
+        .oneOf(monthArr.concat((monthArr.map((mon) => t(mon)))), t('Wrong data format')),
+      year: Yup.number()
+        .required()
+        .min(1920, t('Wrong data format'))
+        .max(2039, t('Wrong data format')),
```

Comment:
why min is 1920 and max 2039?

Reply:
I haven't any requirements about this, but there aren't any dates over this cut, as I see.

Reply:
this is universal component